BUIP-UAHF/buip-uahf-technical-spec.md

```diff
@@ -151,6 +153,9 @@ NOTE 2: The client shall still accept transactions whose signatures
 verify according to pre-fork rules, subject to the additional OP_RETURN
 constraint introduced by REQ-6-1.
 
+NOTE 3: If bit 6 is not set, only the traditional non-shifted nHashType
```

Comment:
Yes, but in the requirement description I did not mean it in the bitshifting sense.
Besides, it can be considered an addition, and my implementation is not authoritative here - this spec is for a new implementation after all.

Change to "If bit 6 is not set, only the traditional *unmodified* nHashType [...]" ?

So I would suggest we change the wording to 

>